compiler/test/runnable/b16278.d

```diff
@@ -1,4 +1,3 @@
-// REQUIRED_ARGS: -main
```

Comment:
why is this a runnable test in the first place?

Reply:
I don't know.